packages/web/src/components/Layout/MarketingLayout.tsx

```diff
@@ -12,9 +12,9 @@ export const MarketingLayout: React.FC = ({ children }) => {
         <GiCrossFlare size="36px" />
         <Heading>Project X</Heading>
         <Spacer />
-        <Link href="/app" passHref>
+        <Link href="/login/github" passHref>
           <Button as="a" size="sm" colorScheme="blue">
-            Go to App
+            Login with Github
```

Comment:
Brands are incredibly important in public-facing applications. Make sure to represent the brand correctly, including spacing and capitalization!

```suggestion
            Login with GitHub
```